action.yml

```diff
@@ -43,6 +49,9 @@ runs:
       shell: bash
       run: |
         (cd ${{ github.action_path }}; FORKLIFT_DIR=$GITHUB_WORKSPACE build_and_setup_everything_bazel.sh)
+      
```

Comment:
nitpicking: I find it more readable to have the bulk of properties of each element together so just for readability, I'd drop this empty line